README.md

```diff
@@ -8,6 +8,7 @@ Docker Tutorials project contains Session PPT's & Lab sessions of Docker Series
 ## Docker for beginners 
    - [Session PPT](docker_for_beginner/docker_for_beginners.pdf)
    - [Lab Session (Basics)](docker_for_beginner/docker_for_beginners_hands_on_session_commands.md)
+   - [Lab Session for Build Docker Image From A Running Container](docker_for_beginner/build_docker_image_from_a_running_container.md)
```

Comment:
It should be **Lab Session - Building docker from running container**

Reply:
done